dbt_sugar/core/task/doc.py

```diff
@@ -94,6 +95,50 @@ def change_model_description(
                     model["description"] = user_input["model_description"]
         return content
 
+    def move_name_and_description_to_first_position(self, model: Dict[str, Any]):
+        """
+        Move keys name and description to the beginning of the dictionary.
+
+        Args:
+            model (Dict[str, Any]): Name of the model to document.
+
+        Returns:
+            Dict[str, Any]: with the keys name and description in the begining of the dictionary.
+        """
+        ordered_dict = OrderedDict(model)
+        ordered_dict.move_to_end("description", last=False)
+        ordered_dict.move_to_end("name", last=False)
```

Comment:
gosh that's such a weird API... I know it's not you though but yeah it's funny.

Can you add a comment on top of line 109 that is the following:
```python
# DEPRECATION: Drop ordered dict when dropping python 3.6 support
```
so that we can relieve ourselves from using an ordered dict when we can?